src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -17,4 +17,7 @@ public class CliSyntax {
     public static final Prefix PREFIX_MODULE = new Prefix("/mod");
     public static final Prefix PREFIX_LECTURE = new Prefix("/lec");
     public static final Prefix PREFIX_VIDEO = new Prefix("/vid");
+
+    /*Flag definitions */
+    public static final Flag FLAG_TAG = new Flag("-t");
```

Comment:
May I know what is the intent of this argument? Seems like it might be breaking the convention.

If the goal is to specify a flag (a value that is either true or false), would it be better to just have a prefix for that flag? For example, if I have a flag for find command that toggles whether I'm searching by tag, would it be better to use a `/searchTag` prefix where the value of the argument is irrelevant (will be ignored)?

Then when parsing the arguments, we can make use of the `ArgumentMultimap.getValue(Prefix)` method to retrieve the `Optional` with the prefix for `/searchTag` and check if it was specified using the `isPresent()` method of `Optional`.

Reply:
As discussed, /byTag prefix will be used to detect if user wants to find by tag or moduleCode/lectureName/videoName.

Have updated my code to follow the convention mentioned thanks!